application/akka/scala/AkkaQuickstart.scala

```diff
@@ -0,0 +1,23 @@
+//#full-example
+package example
+
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+
+//#main-class
+object AkkaQuickstart extends App {
```

Comment:
Coud you rename this, e.g. to `Example`.

Reply:
Done